homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -104,17 +119,20 @@ async def get_tracking_devices(hass: HomeAssistant) -> tuple[set[str], set[str]]
 def lookup_name(mac: str) -> str | None:
     """Lookup a Bluetooth device name."""
     _LOGGER.debug("Scanning %s", mac)
-    return bluetooth.lookup_name(mac, timeout=5)
+    return bluetooth.lookup_name(mac, timeout=5)  # type: ignore[no-any-return]
 
 
 async def async_setup_scanner(
-    hass: HomeAssistant, config: dict, async_see, discovery_info=None
-):
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_see: Callable[..., None],
```

Comment:
If it's truly async.
```suggestion
    async_see: Callable[..., Awaitable[None]],
```